packages/astro/src/core/config/config.ts

```diff
@@ -136,6 +136,23 @@ function mergeCLIFlags(astroConfig: AstroUserConfig, flags: CLIFlags, cmd: strin
 	return astroConfig;
 }
 
+async function search(fsMod: typeof fs, root: string) {
+	const paths = [
+		'astro.config.mjs',
+		'astro.config.js',
+		'astro.config.ts',
+		'astro.config.mts',
+		'astro.config.cjs',
+		'astro.config.cts',
+	].map((p) => path.join(root, p));
+
+	for (const file of paths) {
+		if (fsMod.existsSync(file)) {
```

Comment:
The previous function uses `stat` to check file existence, I switched to `existsSync` since it should be the same.